css/index.css

```diff
@@ -12,9 +12,83 @@ header{
 .navlinks{
     width:500px;
     display:flex;
+    gap: 50px;
+    text-decoration: none;
     justify-content: space-evenly;
     list-style-type: none;
 }
+/*Smaller Screens mobiles
+change the font size (1st change) and color (2nd change) of your headers (1st element), 
+change the padding or margin (1st change) and background color (2nd change) of your unordered lists (2nd element),
+ and change the font size (1st change) 
+and font style (2nd change) of your name at the top of your page (3rd element)*/
+
+@media (min-width:600px){
```

Comment:
So, it looks like you currently have 4 CSS layouts: One for screens smaller than 600px width, one for 600px-992px, one for 992px - 1199px, and one for 1200px or more.

To verify this, open your `index.html` file in a desktop browser window with a width of greater than 1200px and collapse it to invoke the responsiveness. You'll see the current four levels.

Based on the comment above, I don't think this is intended. If you were wanted any device below 600px to use the set of properties in this `media` tag, you'd want to use `max-width:600px`. From there, just adjust the other media properties to use `max-width`.

You could also just remove the media tag from this set of attributes and remove the redundant ones. That way anything under 992 pixels would default to these.

Since it seems you're going with mobile first, I'd staying with the `min-width` tags.